clang-16 complains about casting an integer value that exceeds an
enumeration capacity in the class base::BitField
to not modify chromium code we just cancel warning about this
circumstance to prevent it from being converted to an error that fails
compilation

[sam: Note that this is a bandaid and won't work with Clang 17.]

--- src/3rdparty/chromium/v8/src/base/bit-field.h
+++ src/3rdparty/chromium/v8/src/base/bit-field.h
@@ -39,8 +39,11 @@
   static constexpr int kLastUsedBit = kShift + kSize - 1;
   static constexpr U kNumValues = U{1} << kSize;
 
+  #pragma clang diagnostic push
+  #pragma clang diagnostic ignored "-Wenum-constexpr-conversion"
   // Value for the field with all bits set.
   static constexpr T kMax = static_cast<T>(kNumValues - 1);
+  #pragma clang diagnostic pop
 
   template <class T2, int size2>
   using Next = BitField<T2, kShift + kSize, size2, U>;
